tests/conftest.py

```diff
@@ -262,25 +262,77 @@ def new_epoch():
         casper.initialize_epoch(next_epoch)
     return new_epoch
 
-
 @pytest.fixture
-def mk_validation_code():
-    def mk_validation_code(address):
-        """
-        validation_code = '''
-        ~calldatacopy(0, 0, 128)
-        ~call(3000, 1, 0, 0, 128, 0, 32)
-        return(~mload(0) == {})
-        '''.format(utils.checksum_encode(address))
-        return serpent.compile(validation_code)
-        """
-        # The precompiled bytecode of the validation code which
-        # verifies EC signatures
-        validation_code_bytecode = b"a\x009\x80a\x00\x0e`\x009a\x00GV`\x80`\x00`\x007` "
-        validation_code_bytecode += b"`\x00`\x80`\x00`\x00`\x01a\x0b\xb8\xf1Ps"
-        validation_code_bytecode += address
-        validation_code_bytecode += b"`\x00Q\x14` R` ` \xf3[`\x00\xf3"
-        return validation_code_bytecode
+def wrap_lll_list():
+    def wrap_lll_list(lll_list):
+        return (
+            ['seq',
+                ['return', [0],
+                    ['lll',
+                        lll_list,
+                        [0]
+                    ]
+                ]
+            ]
+        )
+    return wrap_lll_list
+
+
+@pytest.fixture
+def mk_validation_code(wrap_lll_list):
+    def mk_validation_code(address, valcode_type):
+        valcodes_lll_lists = dict(
+            pure=wrap_lll_list(
+                ['seq',
```

Comment:
Here is some repetition, but the readability is no worse than abstracting out.

Reply:
I also considered a "template" style for the LLL, where you specify an impure operation to be inserted at the top of the pure code (i.e., no repetition of actual signing code). I decided against because I thought it might be inflexible... Keen to hear your thoughts though!

Reply:
I think we should make a `tests/utils` folder and move the `valcodes_lll_lists` to a file in there that we import into conftest.py. Either just import the dict or make a function that takes the type of valcode as an argument ("pure", "sload", etc).

The `wrap_lll_list` also doesn't need to be a fixture at this point and should just be a helper function that lives in the utils file.